lib/unison-util-relation/benchmarks/relation/Main.hs

```diff
@@ -1,27 +1,55 @@
-{- ORMOLU_DISABLE -} -- Remove this when the file is ready to be auto-formatted
 module Main where
-import Criterion.Main
-import qualified Unison.Util.Relation as R
-import System.Random
+
 import Control.Monad
+import qualified Data.Set as Set
+import System.Random
+import Test.Tasty.Bench
```

Comment:
Is it much work to convert the other benchmarks to use Tasty too?
As I mentioned in DMs I don't really care which suite we use, but would really really prefer that we use the same one everywhere.

Reply:
It's very easy to change `criterion` to `tasty-bench` because they have the same API, however I also don't think it's a huge issue if we use more than one